addons/web/static/src/core/ui/block_ui.js

```diff
@@ -71,12 +71,10 @@ export class BlockUI extends Component {
 }
 
 BlockUI.template = xml`
-    <div t-att-class="state.blockUI ? 'o_blockUI' : ''">
+    <div t-att-class="state.blockUI ? 'o_blockUI fixed-top d-flex justify-content-center align-items-center flex-column vh-100 bg-black-75' : ''">
```

Comment:
I find the effect too strong.
Could you test with `bg-black-50` and `backdrop-filter: blur(2px);` ?